.github/workflows/builds.yml

```diff
@@ -53,18 +53,20 @@ jobs:
   conda:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
       - uses: conda-incubator/setup-miniconda@v3
         with:
           auto-update-conda: true
           python-version: 3.8
+          channels: pypdfium2-team,bblanchon,conda-forge
```

Comment:
The nice thing with putting the channels directly in the command is that it will be the same for installation & we don't rely too much on the evolution of this Github action.

Reply:
Mh yeah that's correct but now we have both ways with the action and in publish the same thing without (for installation) :)